src/css/styles.scss

```diff
@@ -236,7 +238,22 @@ main {
   flex-wrap: wrap;
 }
 
+.title-container {
```

Comment:
Just checking so I understand, title-container existed but there was no styling for it when we got the project, right?

Reply:
Nope, it didn't exist at all. I added it in the recipe cards so that I could style it with grid and have align and go where I wanted them to go when I added all of the other tags